docs/basic-features/data-fetching.md

```diff
@@ -727,6 +727,12 @@ The `context` parameter is an object containing the following keys:
 >
 > Fetching from an external API is fine!
 
+### Built-in `context.req` parsing
+
+To improve the developer experience, the `context.req` request object received by `getServerSideProps` is pre-parsed to include some additional helpful properties. Those properties are:
+
+- `context.req.cookies` - An object containing the cookies sent by the request. Defaults to `{}`
```

Comment:
@styfle  @leerob @molebox took another stab at this, with a dedicated sub-section for `context.req` built-in parsing. What do you all think?

Reply:
Hmm, I was thinking there were more helpers than just `cookies` but then I checked the type and its just `cookies` like you mentioned 

https://github.com/vercel/next.js/blob/c27e3a41dcf9590fb1c9ba9b9e16517cf4cb49b4/packages/next/types/index.d.ts#L148-L151

And it looks like `res` doesn't have any helpers so maybe its best to go back to your original commit where it was mentioned inline.

Reply:
Or maybe we discovered a strange behavior where `getServerSideProps()` differs to much from API Routes and we should try to use the same helpers for both 🤔

Reply:
Personally, I don't think it hurts to have this extra section, even if it just talks about 1 helper. Nice for google foo

Reply:
I agree with @molebox, I like that we'd be able to deeplink to this section. Additionally, this makes it easier to expand the list in the future if we do want to get to parity with the API route helpers